owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/markup/html/basic/BigDecimalToIntegerLabel.java

```diff
@@ -25,6 +25,12 @@
 import org.apache.wicket.util.convert.IConverter;
 import org.apache.wicket.util.convert.converter.AbstractConverter;
 
+/**
+ * Use {@code new CoreLabel(id, model)} and register a {@code Renderer} for the specific type instead.
+ * Use {@code new CoreLabel(id, MyTypeRenderer.get(...).asModel(model))} to display with a non-default renderer.
+ * @deprecated
```

Comment:
As we discussed together, this comment should be more straightforward: just provide the code one can use to create the renderer.